graphql/database/userImpact/__mocks__/UserImpactModel.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-env jest */
```

Comment:
this is like a very very primitive mock model.  I started toying with a base model mock class but decided to go as simple as possible for the scope of this work then expand later